incubator/graylog/values.yaml

```diff
@@ -0,0 +1,242 @@
+# Default values for Graylog.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+rbac:
+  # Specifies whether RBAC resources should be created
+  ##
+  create: true
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  ##
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  ##
+  name:
+
+tags:
+  # If true, this chart will install Elasticsearch from requirement dependencies
+  install-elasticsearch: true
+  # If true, this chart will install MongoDB replicaset from requirement dependencies
+  install-mongodb: true
+
+graylog:
+  ## Graylog image version
+  ## Ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  image:
+    repository: "graylog/graylog:2.4"
+    pullPolicy: "IfNotPresent"
+
+  replicas: 2
+
+  ## Pod affinity
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ##
+
+  affinity: {}
+  ## Node tolerations for node-exporter scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+
+  tolerations: []
+    # - key: "key"
+    #   operator: "Equal|Exists"
+    #   value: "value"
+    #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+
+  ## Node labels for node-exporter pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## Annotations to be added to Graylog pods
+  ##
+  podAnnotations: {}
+
+  persistence:
+    ## If true, Graylog will create/use a Persistent Volume Claim
+    ## If false, use emptyDir
+    ##
+    enabled: true
+    ## Graylog data Persistent Volume access modes
+    ## Must match those of existing PV or dynamic provisioner
+    ## Ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+    ##
+    accessMode: ReadWriteOnce
```

Comment:
@KongZ you might want to update Persistent Volume to`accessMode: [ "ReadWriteOnce" ]`

Reply:
The template code already declare array. If we use array in value file, it will be broken.
```
      spec:
        accessModes:
          - {{ .Values.graylog.persistence.accessMode | quote }}
```